private-resources/META-INF/maven/gov.usdot/fedgov-cv-private-resources-webapp/pom.properties

```diff
@@ -0,0 +1,5 @@
+#Generated by Apache Maven
+#Thu Dec 21 12:25:17 EST 2017
+version=1.0.0-SNAPSHOT

```

Comment:
Nit: the red `^M` text indicates windows line ending characters. When generating new files sometimes VSCode defaults to this line ending. We try to make all our files use linux line ending characters. To change it for a single file there should be a small option at the bottom of the VSCode editor when you have a file open:
![image](https://github.com/usdot-fhwa-stol/connectedvcs-tools/assets/77466294/68474be9-7935-4cc7-8b26-bbf2b0ab4e42)
If you could just update the couple of files that seem to have incorrect line endings that would be great. Again this is probably non critical so I will leave it up to you.

Reply:
File was not needed, removed.